pages/linux/e2image.md

```diff
@@ -1,7 +1,7 @@
 # e2image
 
 > Save critical ext2/ext3/ext4 filesystem metadata to a file.
-> More information: <https://linux.die.net/man/8/e2image>.
+> More information: <https://man7.org/linux/man-pages/man8/e2image.8.html>.
```

Comment:
Looks like there is `https://man.archlinux.org/man/e2image.8.en`, we can use this one here.

```suggestion
> More information: <https://man.archlinux.org/man/e2image.8.en>.
```